test/powershell/Modules/Microsoft.PowerShell.Management/Get-Process.Tests.ps1

```diff
@@ -18,6 +32,43 @@ Describe "Get-Process" -Tags "CI" {
     It "Should have not empty Name flags set for Get-Process object" -Pending:$IsMacOS {
         $ps | foreach-object { $_.Name | Should Not BeNullOrEmpty }
     }
+
+    It "Should throw an error for non existing process id." {
+        $randomId = 123456789
+        { Get-Process -Id $randomId -ErrorAction Stop } | ShouldBeErrorId "NoProcessFoundForGivenId,Microsoft.PowerShell.Commands.GetProcessCommand"
+    }
+
+    It "Should throw an exception when process id is null." {
+        { Get-Process -id $null } | Should -Throw
+    }
+
+    It "Should throw an exception when -InputObject parameter is null." {
+        { Get-Process -InputObject $null } | Should -Throw
+    }
+
+    It "Should not fail to get process name even if it is unavailable." {
+        { (Get-Process -Id $idleProcessPid).Name } | Should -Not -Throw
```

Comment:
I suppose we should avoid `Should -Not -Throw` -it is better to check the expected positive result (-BeNullOrEmpty)

Reply:
Yes, but for instance the Idle process seems to not have name property only on Unix system, on Widnows there is a name. I can maybe skip the test for Windows as well, but in this way it is more generic.

Reply:
If we return different results on different platforms it is bug.